Framework/src/Framework/Process.cxx

```diff
@@ -484,8 +481,10 @@ void Process::newRun(ldmx::RunHeader &header) {
   if (performance_) performance_->stop(performance::Callback::onNewRun, 0);
 }
 
-bool Process::process(int n, Event &event) const {
-  if ((logFrequency_ != -1) && ((n + 1) % logFrequency_ == 0)) {
+bool Process::process(int n, int n_try, Event &event) const {
+  if ((logFrequency_ != -1) && ((n + 1) % logFrequency_ == 0) && (n_try < 2)) {
+    // only printout event counter if we've enabled log frequency, the event
+    // matches the frequency and we are on the first try
```

Comment:
this change reminds me, i've grown used to having to set the termLogLevel to 1 or less to get event number printouts even with logFrequency set to something reasonable (like 10 printouts per job). does that change in the current implementation?